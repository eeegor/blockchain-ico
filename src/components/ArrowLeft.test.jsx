import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { ArrowLeft } from '.';

describe('ArrowLeft', () => {
	test('renders ArrowLeft', () => {
		const tree = renderer.create(<ArrowLeft />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
