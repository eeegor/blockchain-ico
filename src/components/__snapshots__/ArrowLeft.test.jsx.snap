// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowLeft renders ArrowLeft 1`] = `
<svg
  height={10}
  version="1.1"
  viewBox="0 0 10 10"
  width={10}
  xmlns="http://www.w3.org/2000/svg"
  xmlnsXlink="http://www.w3.org/1999/xlink"
>
  <g
    fill="none"
    fillRule="evenodd"
    id="arrow-left"
    stroke="none"
    strokeWidth="1"
  >
    <polygon
      fill="#2B60FF"
      fillRule="nonzero"
      points="5.7315513 1.28422588 9.44903182 5.0017064 5.7315513 8.71918692 3.28115987 8.71918692 6.10606916 5.89427763 0.550968178 5.89076591 0.550968178 4.09157656 6.08860947 4.09167548 3.27774707 1.28081308"
    />
  </g>
</svg>
`;
